python/lsst/meas/algorithms/utils.py

```diff
@@ -737,14 +744,17 @@ def showPsfMosaic(exposure, psf=None, nx=7, ny=None,
         i = 0
         with ds9.Buffering():
             for cen, shape in zip(centers, shapes):
-                bbox = mos.getBBox(i); i += 1
+                bbox = mos.getBBox(i)
+                i += 1
                 xc, yc = cen[0] + bbox.getMinX(),  cen[1] + bbox.getMinY()
                 if showCenter:
                     ds9.dot("+", xc, yc,  ctype=ds9.BLUE, frame=frame)
 
                 if showEllipticity:
                     ixx, ixy, iyy = shape
-                    ixx *= scale; ixy *= scale; iyy *= scale
```

Comment:
@RobertLuptonTheGood likes putting these kinds of things on one line, since they go together.  I think you need to fight it out with him.